source/common/network/connection_impl.cc

```diff
@@ -722,7 +723,8 @@ ClientConnectionImpl::ClientConnectionImpl(
 
 void ClientConnectionImpl::connect() {
   ENVOY_CONN_LOG(debug, "connecting to {}", *this, socket_->remoteAddress()->asString());
-  const Api::SysCallIntResult result = socket_->remoteAddress()->connect(ioHandle().fd());
+  // const Api::SysCallIntResult result = socket_->remoteAddress()->connect(ioHandle().fd());
```

Comment:
?

Reply:
.. just me not being careful. Sorry about that.